src/instructlab/configuration.py

```diff
@@ -388,6 +388,12 @@ class _train(BaseModel):
         protected_namespaces=(),
         use_enum_values=True,  # populate models with the value property of enums, rather than the raw enum.
     )
+    pipeline: str = Field(
+        default="accelerated",
```

Comment:
Should we default to `accelerated` you think, or `full`?

Reply:
For the open source project, full is a better choice. For the downstream, maybe accelerated is better.

Reply:
yeah we can make the pydantic default full and then put accelerated in the d/s config file